modules/ROOT/pages/bolt/message.adoc

```diff
@@ -742,16 +757,57 @@ Valid values for `api` and the corresponding API associations are:
 - `2` -- Implicit transaction
 - `3` -- Driver-level `execute_query()`
 
+*Detail messages:*
+
+No detail messages should be returned.
+
+*Valid summary messages:*
+
+** `SUCCESS`
+** `FAILURE`
+
 ==== Synopsis
 ----
```

Comment:
```suggestion
[source, syntax]
----
```

Reply:
Basically none of the synopsis or example sections have this. I suggest to keep it somewhat consistent by not introducing it now and only here but later and everywhere. Wdyt?

Reply:
Yeah fair enough!

Reply:
I created an issue (#43) to take care of that in a separate PR.